scripting/nexus-script-example/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'groovy'
```

Comment:
Yeah, on second glance, the Plugin DSL is still in incubating state and `apply` hasn't been deprecated.